docs/reference/migration/migrate_8_0/search.asciidoc

```diff
@@ -6,6 +6,20 @@
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[[_type-search-matches-no-docs]]
+.A search on the `_type` field now matches no documents.
```

Comment:
I think a clearer way to phrase this would be "searches on `_type` are no longer supported". We actually completely removed support for searching on `_type` -- it's not recognized as a valid field. It just so happens that searches on non-existent fields will return no results. This note makes might suggest that `_type` is still a valid metadata field, whereas it doesn't exist anymore!

Reply:
Thanks @jtibshirani. I added some additional context with 581393ed8be92e3b6273ac5d808e73f97c08b2b5.